activemq-karaf/src/main/resources/features-core.xml

```diff
@@ -31,6 +31,7 @@
         <bundle dependency="true">mvn:org.jvnet.jaxb2_commons/jaxb2-basics-runtime/${jaxb-basics-version}</bundle>
         <bundle dependency='true'>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.jaxb-impl/${jaxb-bundle-version}</bundle>
         <bundle>mvn:org.apache.commons/commons-pool2/${commons-pool2-version}</bundle>
+        <bundle dependency="true">mvn:org.apache.zookeeper/zookeeper/${zookeeper-version}</bundle>
```

Comment:
@jbonofre - this looks wrong, I think this might have come back by mistake during the merge?

Reply:
This was a clean-up in AMQ-7309. Zookeeper was removed in a prior change, so this should NOT be added back in